### PR TITLE
Removed msedge-canary, because not supported by Playwright

### PR DESCRIPTION
**Pull request by kumardeepakr3**
As per docs of playwright only msedge-dev and msedge-beta are the other supported channels apart from stable.

**Rendered section for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/playwright/?branch=pr-en-us-1686#run-tests-in-microsoft-edge

### CODE REVIEW

**Comment by mikehoffms** [comment]
> per docs of Playwright, only msedge-dev and msedge-beta are the other supported channels, apart from stable

Is there a URL for the Playwright docs where it states that only Stable, Dev, and Beta are supported, not Canary?

File: microsoft-edge/playwright/index.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ const config: PlaywrightTestConfig = {
     {
       name: 'Microsoft Edge',
       use: {
-        // Supported Microsoft Edge channels are: msedge, msedge-beta, msedge-dev, msedge-canary
+        // Supported Microsoft Edge channels are: msedge, msedge-beta, msedge-dev
```
Comment (mikehoffms) [inline microsoft-edge/playwright/index.md:79]: Is there a URL for the Playwright docs where it states that only Stable, Dev, and Beta are supported, not Canary?

Comment (mikehoffms) [reply microsoft-edge/playwright/index.md:79]: @kumardeepakr3 @bwalderman @zoherghadyali - Is there a URL for the Playwright docs where it states that only Stable, Dev, and Beta are supported, not Canary?

Comment (bwalderman) [reply microsoft-edge/playwright/index.md:79]: I can't find where this is stated in the https://playwright.dev docs. The API docs show that `"msedge-canary"` is supported. E.g.: https://playwright.dev/docs/api/class-testoptions#test-options-channel

Comment (kumardeepakr3) [reply microsoft-edge/playwright/index.md:78]: The step that directly follows the code snippet fails for 'msedge-canary'. Running 'npx playwright install msedge-canary' throws error. However when someone installs canary directly and uses it in the code snippet it works. My bad, abandoning the PR.

image

**Pull request closed without merging**